app/src/main/java/xyz/jhughes/laundry/LaundryParser/Machine.java

```diff
@@ -21,7 +23,10 @@ public Machine(String name, String type, String status, String time) {
     }
 
     public String getName() {
-        return name;
+        // Get rid of leading 0s
+        if (name.matches(".*0[1-9][0-9].*")) return name.replaceFirst("0", "");
+        else if (name.matches(".*0{2}[1-9].*")) return name.replaceAll("0", "");
```

Comment:
The server is on AWS with Docker and Redis and all that. We even have reports that come back with status occasionally. 

We have access to the AWS, we just need to actually make the change, and I am not entirely sure how to be honest. But we do have the required control.